### ISSUE TITLE
Choice of License

### ISSUE THREAD

**Issue by xiw315** [UBC-MDS/DSCI_524_Group2_Python#90]
This issue is to discuss whether our current license is appropriate and what other license we might want to consider for our package.

**Comment by andrealee011** [comment]
I think we can keep our license as is (i.e. MIT license). Because we want to be able to openly distribute our package without restriction, an MIT license would be appropriate.

We could also consider a GNU General Public License. This is similar to a MIT license, except those who use our package must also make their source code openly available. This will ensure no one is profiting off our package.

**Comment by eyrakas** [comment]
Agree. We can stick to MIT License.

**Comment by xiw315** [comment]
I agree that we keep using MIT license. Other the reason Andrea already stated, MIT license is probably one of the most widely used licenses so potential contributors and users should all be pretty familiar with its terms. Thus, they do not need to spend much time deciding whether they will be okay with our license.

**Comment by agdal1125** [comment]
I also agree with keepin the MIT license. As Xinwen mentioned, it is the most well known licenses
